src/components/App/App.js

```diff
@@ -1,38 +1,41 @@
 import React, { Component } from 'react';
 import queries from '../../utils/queries/queries';
+import { LoginForm } from '../LoginForm/LoginForm';
```

Comment:
Interesting thing, I thought that when importing a component that was connected to Redux, we only used the named import to import into the test and the default import when importing into other components. However, the linter was asking me to use a named import here and it did not cause any problems for me. I have definitely had issues with trying to use named imports in the past on a connected component. Any insight?

Reply:
I personally haven't had issues with named imports with components connected to the store, let's try it out. If we run into issues we can just edit the rule in the linter